source/localizable/feedspec/index.en.html.md.erb

```diff
@@ -361,6 +374,23 @@ The color of the product
 ~~~
 
 
+<% if settings.feed_validator_domain %>
+## Data Feed Validator
+
+You can use our public <%= link_to "Data Feed Validator",
+"http://#{settings.feed_validator_domain}" %> to check your XML feed for
+compatibility with <%= settings.site_name.capitalize %> standards.
+
+There, you can upload your XML file and the validator checks to confirm that it is
+well-formed according to the XML specification, sees if it can identify the XML
+element corresponding to each one of your products and then scans the products'
+attributes to ensure that <%= settings.site_name.capitalize %> requirements are met.
+
+<%= link_to image_tag("#{settings.site_name}/validator.png", class: 'img-responsive'),
+      "http://#{settings.feed_validator_domain}"  %>
+<% end %>
```

Comment:
I actually prefer the image below the text, as it functions like a "call to action" for readers, after they have finished reading the above. Otherwise it is out of context, the user doesn't necessarily know what that image refers to. 

Also the centered image looks kind of misplaced, as the text is left-aligned.